packages/tasit-action/src/Contract.test.js

```diff
@@ -311,7 +311,7 @@ describe("TasitAction.Contract", () => {
       //  https://github.com/sinonjs/sinon/issues/1739
       //  https://github.com/sinonjs/lolex/issues/114
       //  https://stackoverflow.com/a/50785284
-      await wait(txSubscription.getEventsTimeout() * 2);
+      await wait(txSubscription.getEventsTimeout() * 3);
```

Comment:
Out of curiosity, why did this prove to be useful?

Reply:
Yes, that's working. I've raised the value because one of tons tests failed here.

Reply:
The test failed on expectation `expect(errorFn.called).to.be.true;` so error event wasn't triggered. Since it's a time based event, the only explanation that surges to me is that time waiting wasn't enough. I'm not sure why it happened.

Reply:
Good to know. Thanks for the explanation.